### ISSUE TITLE
Multiline comments for free (nearly)

### ISSUE THREAD

**Issue by kasperosterbye** [pillar-markup/Microdown#409]
At present we have `%` as a single line comment. Sometimes I find it useful to comment out a piece of markdown, or insert a longer explanation inline in the microdown source which should not be rendered. Prefixing each line with `%` is cumbersome.

However, as we now have extension on the MathBlock `&...&`, I propose an extension:
```text
&comment
bla bla
more bla
&
```
This extension can then be rendered as nothing making it a de-facto comment. It requires a `MicMultilineComment` as subclass of `MicMathBlock` with an accept which does nothing.

**Comment by kasperosterbye** [comment]
I did try it out, and it was rather simple. However, a somewhat more elaborate solution using an environment which do not parse it body is more attractive as it will function as a comment in github as well.
<!COMMENT
this is a comment
!>
```
<!COMMENT
this is a comment
!>
```

I did run into a need for multiline comments in some of my Microdown documentation where I wanted to save some todos and unfinished parts for later.
